py3status/modules/whatismyip.py

```diff
@@ -13,14 +14,13 @@
         (default 'ip')
     negative_cache_timeout: how often to check again when offline (default 2)
     timeout: how long before deciding we're offline (default 5)
-    url: change IP check url (must output a plain text IP address)
-        (default 'http://ultrabug.fr/py3status/whatismyip')
```

Comment:
i'm not sure how @ultrabug will be about you removing his wonderful service.  I think if we only are looking for `{ip}` then we could continue to use this.

How about:

* deprecate url_geo and default `url = None`

* if user supplies url we use it

* no supplied url then default to ultrabug.fr if only ip is wanted else freegeoip.net

* when we call request the url we can see if it is json or just an ip and act accordingly